--- a/apps/organizations_ext/invitation_backend.py	2023-01-20 14:45:54
+++ b/apps/organizations_ext/invitation_backend.py	2023-01-20 14:46:03
@@ -66,11 +66,4 @@
         return InvitationTokenGenerator().make_token(org_user)

     def send_invitation(self, user, sender=None, **kwargs):
-        kwargs.update(
-            {
-                "token": self.get_token(user),
-                "organization": user.organization,
-            }
-        )
-        send_email_invite.delay(user.pk, kwargs["token"])
         return True
