diff --git a/apps/organizations_ext/api.py b/apps/organizations_ext/api.py
index 3173d0ac..5bf2c957 100644
--- a/apps/organizations_ext/api.py
+++ b/apps/organizations_ext/api.py
@@ -245,10 +245,31 @@ async def create_organization_member(
     if teams:
         await member.teams.aadd(*teams)
 
-    await sync_to_async(invitation_backend().send_invitation)(member)
-    member = await get_organization_users_queryset(user_id, organization_slug).aget(
-        id=member.id
+    # automatically create and connect Django user and Glitchtip organization user
+    from django.contrib.auth import get_user_model
+    from allauth.account.models import EmailAddress
+
+    user_model = get_user_model()
+    user, created = await user_model.objects.aget_or_create(
+        email=email,
+        defaults={
+            "password": "!",
+            "is_active": True,
+        },
+    )
+    if created:
+        await EmailAddress.objects.aget_or_create(
+            user=user, email=email, primary=True, verified=True
+        )
+    # Fetch user obj from DB after creation to prefetch socialaccount_set. Needed for API response
+    user = await user_model.objects.prefetch_related("socialaccount_set").aget(
+        email=email
     )
+
+    member.user = user
+    member.email = None
+    await member.asave()
+
     return 201, member
 
 
