diff --git a/glitchtip/social.py b/glitchtip/social.py
index 8fe4f75c..a5d270ef 100644
--- a/glitchtip/social.py
+++ b/glitchtip/social.py
@@ -190,6 +190,14 @@ class SocialLoginSerializer(BaseSocialLoginSerializer):
                     ) from ex
                 self.post_signup(login, attrs)
 
+        # automatically accept any open invitation for this user
+        from apps.organizations_ext.models import OrganizationUser
+
+        for org_user in OrganizationUser.objects.filter(email=login.user.email):
+            org_user.user = login.user
+            org_user.email = None
+            org_user.save()
+
         attrs["user"] = login.account.user
 
         return attrs
