diff --git a/alerts/tasks.py b/alerts/tasks.py
index e008668..5c9bbcf 100644
--- a/alerts/tasks.py
+++ b/alerts/tasks.py
@@ -27,9 +27,9 @@ def process_event_alerts():
         issues = (
             Issue.objects.filter(
                 project_id=alert.project_id,
-                notification__isnull=True,
                 event__created__gte=start_time,
             )
+            .exclude(notification__project_alert=alert)
             .annotate(num_events=Count("event"))
             .filter(num_events__gte=quantity_in_timespan)
         )
