diff --git a/glitchtip/api/api.py b/glitchtip/api/api.py
index 898bffe..c967270 100644
--- a/glitchtip/api/api.py
+++ b/glitchtip/api/api.py
@@ -1,7 +1,7 @@
 from typing import Optional
 
 from allauth.socialaccount.models import SocialApp
-from allauth.socialaccount.providers.microsoft.views import MicrosoftGraphOAuth2Adapter
+from allauth.socialaccount.providers.oauth2.views import OAuth2Adapter
 from allauth.socialaccount.providers.openid_connect.views import OpenIDConnectAdapter
 from asgiref.sync import sync_to_async
 from django.conf import settings
@@ -79,6 +79,10 @@ class SettingsOut(CamelSchema):
     server_time_zone: str
 
 
+def get_authorize_url(adapter: OAuth2Adapter) -> str:
+    return adapter.authorize_url
+
+
 @api.get("settings/", response=SettingsOut, by_alias=True, auth=None)
 async def get_settings(request: HttpRequest):
     social_apps: list[SocialApp] = []
@@ -94,12 +98,7 @@ async def get_settings(request: HttpRequest):
         else:
             adapter = None
         if adapter:
-            if isinstance(adapter, MicrosoftGraphOAuth2Adapter):
-                social_app.authorize_url = await sync_to_async(
-                    adapter._build_tenant_url
-                )("/oauth2/v2.0/authorize")
-            else:
-                social_app.authorize_url = adapter.authorize_url
+            social_app.authorize_url = await sync_to_async(get_authorize_url)(adapter)
 
         social_app.provider = social_app.provider_id or social_app.provider
         social_apps.append(social_app)
