diff --git a/issues/migrations/sql/functions.py b/issues/migrations/sql/functions.py
index 983f5bf..27e979e 100644
--- a/issues/migrations/sql/functions.py
+++ b/issues/migrations/sql/functions.py
@@ -98,7 +98,7 @@ SET
   last_seen = GREATEST(event_agg.new_last_seen, issues_issue.last_seen),
   level = GREATEST(event_agg.new_level, issues_issue.level),
   search_vector = CASE WHEN pg_column_size(COALESCE(search_vector, ''::tsvector)) < 500000 THEN concat_tsvector(COALESCE(search_vector, ''::tsvector), event_vector.vector) ELSE search_vector END,
-  tags = CASE WHEN pg_column_size(tags) < 10000000 THEN COALESCE(jsonb_merge_deep(event_agg.new_tags, tags), '{}') ELSE tags END
+  tags = CASE WHEN pg_column_size(tags) < 500000 THEN COALESCE(jsonb_merge_deep(event_agg.new_tags, tags), '{}') ELSE tags END
 FROM event_agg, event_vector
 WHERE issues_issue.id = update_issue_id;
 $$;
