diff --git a/apps/stats/schema.py b/apps/stats/schema.py
index 61cffa4e..d6de0622 100644
--- a/apps/stats/schema.py
+++ b/apps/stats/schema.py
@@ -4,12 +4,13 @@ from typing import Literal
 from ninja import Schema
 from ninja.errors import ValidationError
 from pydantic import Field, model_validator
+from typing_extensions import Annotated
 
 
 class StatsV2Schema(Schema):
     category: Literal["error", "transaction"]
     interval: Literal["1d", "1h", "1m"] | None = "1h"
-    project: list[int] | None = Field(default=None, ge=-1)
+    project: list[Annotated[int, Field(ge=-1)]] | None = None
     field: Literal["sum(quantity)", "sum(times_seen)"]
     start: datetime
     end: datetime
