diff --git a/glitchtip/settings.py b/glitchtip/settings.py
index 6dc45311..c7b36d0e 100644
--- a/glitchtip/settings.py
+++ b/glitchtip/settings.py
@@ -86,6 +86,7 @@ if GLITCHTIP_URL.scheme not in ["http", "https"]:
 TESTING = len(sys.argv) > 1 and sys.argv[1] == "test"
 
 DATA_UPLOAD_MAX_MEMORY_SIZE = 4294967295  # TMP REMOVE THIS
+DATA_UPLOAD_MAX_NUMBER_FIELDS = env.int("DATA_UPLOAD_MAX_NUMBER_FIELDS", default=1000)
 # Limits size (in bytes) of uncompressed event payloads. Mitigates DOS risk.
 GLITCHTIP_MAX_UNZIPPED_PAYLOAD_SIZE = env.int(
     "GLITCHTIP_MAX_UNZIPPED_PAYLOAD_SIZE", global_settings.DATA_UPLOAD_MAX_MEMORY_SIZE
