diff --git a/glitchtip/wsgi.py b/glitchtip/wsgi.py
index 6bf0b5d1..0b13dfab 100644
--- a/glitchtip/wsgi.py
+++ b/glitchtip/wsgi.py
@@ -12,6 +12,16 @@ import os
 from django.core.wsgi import get_wsgi_application
 from uwsgi_chunked import Chunked
 
+try:
+    import prometheus_client
+    import uwsgi
+
+    prometheus_client.values.ValueClass = prometheus_client.values.MultiProcessValue(  # type: ignore
+        process_identifier=uwsgi.worker_id
+    )
+except ImportError:
+    pass  # not running in uwsgi
+
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "glitchtip.settings")
 
 application = Chunked(get_wsgi_application())
