diff --git a/apps/observability/metrics.py b/apps/observability/metrics.py
index dd78c5d5..fdfd7d9f 100644
--- a/apps/observability/metrics.py
+++ b/apps/observability/metrics.py
@@ -1,5 +1,5 @@
 from django.core.cache import cache
-from django.db.models import Count
+from django.db.models import Count, Sum
 from prometheus_client import Gauge
 
 from apps.observability.constants import OBSERVABILITY_ORG_CACHE_KEY
@@ -9,10 +9,14 @@ organizations_metric = Gauge("glitchtip_organizations", "Number of organizations
 projects_metric = Gauge(
     "glitchtip_projects", "Number of projects per organization", ["organization"]
 )
+events_counter = Gauge(
+    "glitchtip_events", "Events per project", ["project", "organization"]
+)
 
 
 async def compile_metrics():
     """Update and cache the organization and project metrics"""
+    from apps.projects.models import IssueEventProjectHourlyStatistic
 
     orgs = cache.get(OBSERVABILITY_ORG_CACHE_KEY)
     if orgs is None:
@@ -28,3 +32,14 @@ async def compile_metrics():
         projects_metric.labels(org["slug"]).set(org["projects__count"])
 
     organizations_metric.set(len(orgs))
+    async for i in (
+        IssueEventProjectHourlyStatistic.objects.select_related(
+            "project", "project__organization"
+        )
+        .values("project__slug", "project__organization__slug")
+        .annotate(events=Sum("count"))
+    ):
+        events_counter.labels(
+            project=i["project__slug"],
+            organization=i["project__organization__slug"],
+        ).set(i["events"])
