diff --git a/glitchtip/ingest_asgi.py b/glitchtip/ingest_asgi.py
index 71aa25d2..a03fce70 100644
--- a/glitchtip/ingest_asgi.py
+++ b/glitchtip/ingest_asgi.py
@@ -48,9 +48,11 @@ class IngestDispatcher:
                 @classmethod
                 def _get_middleware_setting(cls):
                     return [
+                        "django_prometheus.middleware.PrometheusBeforeMiddleware"
                         "django.middleware.security.SecurityMiddleware",
                         "corsheaders.middleware.CorsMiddleware",
                         "glitchtip.middleware.DecompressBodyMiddleware",
+                        "django_prometheus.middleware.PrometheusAfterMiddleware",
                     ]
 
                 def load_middleware(self, is_async=True):
