diff --git a/apps/observability/metrics.py b/apps/observability/metrics.py
index bf01fe0f..08a94ee0 100644
--- a/apps/observability/metrics.py
+++ b/apps/observability/metrics.py
@@ -1,26 +1,46 @@
 from django.core.cache import cache
-from django.db.models import Count
+from django.db.models import Count, Sum
 from prometheus_client import Gauge
 
 from apps.observability.constants import OBSERVABILITY_ORG_CACHE_KEY
 from apps.organizations_ext.models import Organization
 
 organizations_metric = Gauge("glitchtip_organizations", "Number of organizations")
-projects_metric = Gauge("glitchtip_projects_total", "Total number of projects")
+projects_metric = Gauge(
+    "glitchtip_projects", "Number of projects per organization", ["organization"]
+)
+
+events_counter = Gauge(
+    "glitchtip_events", "Events per project", ["project", "organization"]
+)
 
 
 async def update_metrics():
     """Update and cache the organization and project metrics"""
-    counts = await cache.aget(OBSERVABILITY_ORG_CACHE_KEY)
-    if counts is None:
-        result = await Organization.objects.aaggregate(
-            org_count=Count("id"), project_count=Count("projects")
+    from apps.projects.models import IssueEventProjectHourlyStatistic
+
+    orgs = await cache.aget(OBSERVABILITY_ORG_CACHE_KEY)
+    if orgs is None:
+        orgs = [
+            org
+            async for org in Organization.objects.annotate(Count("projects"))
+            .values("slug", "projects__count")
+            .all()
+        ]
+        await cache.aset(OBSERVABILITY_ORG_CACHE_KEY, orgs, 60 * 60)
+
+    for org in orgs:
+        projects_metric.labels(org["slug"]).set(org["projects__count"])
+    organizations_metric.set(len(orgs))
+
+    async for i in (
+        IssueEventProjectHourlyStatistic.objects.select_related(
+            "project", "project__organization"
         )
-        counts = {
-            "org_count": result["org_count"] or 0,
-            "project_count": result["project_count"] or 0,
-        }
-        await cache.aset(OBSERVABILITY_ORG_CACHE_KEY, counts, 60 * 60)
-
-    organizations_metric.set(counts["org_count"])
-    projects_metric.set(counts["project_count"])
+        .values("project__slug", "project__organization__slug")
+        .annotate(events=Sum("count"))
+    ):
+        events_counter.labels(
+            project=i["project__slug"],
+            organization=i["project__organization__slug"],
+        ).set(i["events"])
