diff --git a/glitchtip/social.py b/glitchtip/social.py
index c4ef995..35596b1 100644
--- a/glitchtip/social.py
+++ b/glitchtip/social.py
@@ -159,7 +159,13 @@ class SocialLoginSerializer(BaseSocialLoginSerializer):
                 raise serializers.ValidationError(_("User registration is closed."))
             else:
                 login.lookup()
-                login.save(request, connect=True)
+                login.save(request, connect=False)
+
+        # # automatically accept any open invitation for this user
+        # from organizations_ext.models import OrganizationUser
+
+        # for org_user in OrganizationUser.objects.filter(email=login.user.email):
+        #     org_user.accept_invite(login.user)
 
         attrs["user"] = login.account.user
 
